docs/reference/components/sources/docker_logs.cue

```diff
@@ -84,6 +95,51 @@ components: sources: docker_logs: {
 	}
 
 	configuration: {
+		docker_host: {
+			common: true
+			description: """
+				The Docker host to connect to. Use an HTTPS URL to enable TLS encryption.
```

Comment:
Would it make sense to call out that the default ('null') will result in it automatically trying `/var/run/docker.sock` on unix or `\\.\pipe\docker_engine` on Windows?

I'd also call out that this will default to `$DOCKER_HOST`.